client/my-sites/plans/current-plan/main.jsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal Dependencies
+ */
+import Main from 'components/main';
+import Card from 'components/card';
+import HappinessSupport from 'components/happiness-support';
+import PremiumPlanDetails from 'my-sites/upgrades/checkout-thank-you/premium-plan-details';
+import BusinessPlanDetails from 'my-sites/upgrades/checkout-thank-you/business-plan-details';
+import CheckoutThankYouFeaturesHeader from 'my-sites/upgrades/checkout-thank-you/features-header';
+import { getPlansBySite } from 'state/sites/plans/selectors';
+import {
+	isBusiness,
+	isPremium
+} from 'lib/products-values';
+import Gridicon from 'components/gridicon';
+
+const PlanDetailsComponent = React.createClass( {
+	PropTypes: {
+		selectedSite: React.PropTypes.object.isRequired,
+		sitePlans: React.PropTypes.object.isRequired
+	},
+	render: function() {
+		const { hasLoadedFromServer } = this.props.sitePlans;
+		let featuresList;
+		let title;
+		let tagLine;
+
+		if ( isPremium( this.props.selectedSite.plan ) ) {
+			title = this.translate( 'Your site is on a Premium plan' );
+			tagLine = this.translate( 'Unlock the full potential of your site with the premium features included in your plan.' );
+			featuresList = (
+				<PremiumPlanDetails
+					selectedSite={ this.props.selectedSite }
+					sitePlans={ this.props.sitePlans }
+				/>
+			);
+		} else if ( isBusiness( this.props.selectedSite.plan ) ) {
+			title = this.translate( 'Your site is on a Business plan' );
+			tagLine = this.translate( 'Your site is serious now. Take advantage of these professional features included in a Business plan:' );
+			featuresList = (
+				<BusinessPlanDetails
+					selectedSite={ this.props.selectedSite }
+					sitePlans={ this.props.sitePlans }
+				/>
+			);
+		}
+
+		return (
+			<Main className="current-plan">
+				<Card>
+					<div className="current-plan__header">
+						<div className="current-plan__header-content">
+							<span className="current-plan__header-icon">
+								<Gridicon icon="star" size={ 48 } />
+							</span>
+
+							<div className="current-plan__header-copy">
+								<h1 className="current-plan__header-heading">
+									{ title }
+								</h1>
+
+								<h2 className="current-plan__header-text">
+									{ tagLine }
+								</h2>
+							</div>
+						</div>
+					</div>
+					<CheckoutThankYouFeaturesHeader
```

Comment:
If this component is going to be used outside the context of checkout then we should probably move it to the components directory and give it a new name.